specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-03-01-preview/WorkspaceManagerAssignments.json

```diff
@@ -0,0 +1,734 @@
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceManagerAssignmentName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/workspaceManagerAssignments/{workspaceManagerAssignmentName}/jobs": {
+      "get": {
+        "x-ms-examples": {
+          "Get all jobs for the specified Sentinel workspace manager assignment.": {
+            "$ref": "./examples/workspaceManagerAssignments/GetAllJobs.json"
+          }
+        },
+        "tags": [
+          "workspaceManagerAssignments"
+        ],
+        "description": "Get all jobs for the specified workspace manager assignment",
+        "operationId": "WorkspaceManagerAssignmentJobs_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceManagerAssignmentName"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/ODataOrderBy"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/ODataTop"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/ODataSkipToken"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/JobList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      },
+      "post": {
```

Comment:
>post [](http://example.com/codeflow?start=7&length=4)

post MUST be declared as an action and not on the resource itself

Reply:
Hi! Thanks for the review. Quick question, does this apply to ARM APIs only?
From my understanding a POST operation would happen at the collection level.
https://restfulapi.net/rest-put-vs-post/

Reply:
POST actions in ARM can only be at resource provider scope or at an individual resources scope.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#resource-action-requests

Reply:
Thanks for the information. We currently have customers in preview using this POST API. We spoke about how given it is being used this should be fine for now. Please let me know if this is not still the case :) 
Thanks!

Reply:
The manifest PR that introduced this type should have never been approved since the swagger PRs weren't reviewed.  Unfortunately, it was and this API is already public so letting it stay as is to avoid breaking existing customers